tools/enhancements/metadata.go

```diff
@@ -43,3 +44,79 @@ func (s *Summarizer) GetMetaData(pr int) (*MetaData, error) {
 	}
 	return NewMetaData(content)
 }
+
+// Validate returns a list of issues found with the metadata
+func (m *MetaData) Validate() []string {
+	results := []string{}
+
+	reportError := func(msg string) {
+		results = append(results, msg)
+	}
+
+	// Must have one valid tracking link and no TBD
+	foundLink := false
+	for _, trackingLink := range m.TrackingLinks {
+		if trackingLink == "TBD" {
+			reportError("'TBD' is not a valid value for tracking-link")
+			continue
+		}
+		if trackingLink == "" {
+			reportError("empty string for tracking-link")
+			continue
+		}
+		if u, err := url.Parse(trackingLink); err != nil {
+			reportError(fmt.Sprintf("could not parse tracking-link %q: %s",
+				trackingLink, err,
+			))
+			continue
+		} else {
+			if u.Scheme == "" {
+				reportError(fmt.Sprintf("could not parse tracking-link %q",
+					trackingLink,
+				))
+				continue
+			}
+		}
+		foundLink = true
+	}
+	if !foundLink {
+		reportError("no valid URL found in tracking-link")
+	}
+
+	// No TBD in required people fields
+	for _, field := range []struct {
+		Name string
+		Data []string
+	}{
+		{
+			Name: "authors",
+			Data: m.Authors,
+		},
+		{
+			Name: "reviewers",
+			Data: m.Reviewers,
+		},
+		{
+			Name: "approvers",
+			Data: m.Approvers,
+		},
+		{
+			Name: "api-approvers",
+			Data: m.APIApprovers,
+		},
+	} {
+		valid := 0
+		for _, value := range field.Data {
+			if value == "TBD" {
```

Comment:
Can't we check if this is a valid GitHub id?

Reply:
We can, but doing that reliably would require giving the job a github token (to avoid API load limits) and specifying how to parse the name out of the comment. I didn't want to get into that level of complexity. I trust reviewers to read the values that are there, so this check is just ensuring someone provides a value.

Reply:
Done